### ISSUE TITLE
Error in regex causes avatar thumbnail to break

### ISSUE THREAD

**Issue by jamesa08** [tersiswilvin/Roblox-2019-Old-Theme#43]
### Are you up-to-date?

- [X] I've confirmed that I am completely up-to-date.

### Is the problem in Experimental?

- [X] I've verified that the problem persists in the Experimental Branch.

### Browser

Google Chrome

### Page URL

https://www.roblox.com/home

### Channel

Experimental

### Type

JavaScript

### Describe the Problem

Error in the regex selection, causing the avatar thumbnail to break

line 2486
```js
if (window.devicePixelRatio > 1) {
  avatarThumb.setAttribute("src", avatarThumb.getAttribute("src").replace(/150/g, "720"))
} else {
  avatarThumb.setAttribute("src", avatarThumb.getAttribute("src").replace(/720/g, "150"))
}
```
example URL: https://tr.rbxcdn.com/30DAY-AvatarHeadshot-3A27B04FA26C03150D0F12D3DE3CE9DF-Png/150/150/AvatarHeadshot/Png/noFilter
Currently, it matches everything with 150/720, which will break the avatar hash earlier in the URL
<img width="796" alt="image" src="https://github.com/tersiswilvin/Roblox-2019-Old-Theme/assets/79052050/fd2dbca7-339f-4d4c-917f-ddad18f4b7f9">
This can be fixed by matching with a leading/trailing slash (`/150\//g`) and subsequently replacing it in the replace text.

**Comment by tersiswilvin** [comment]
Hi, thank you for reporting this issue. I'll get around to fixing this later today when I have time.

**Comment by tersiswilvin** [comment]
Fixed in [a6b5631](https://github.com/tersiswilvin/Roblox-2019-Old-Theme/commit/a6b5631196500d2e6de131a46a5bbca3fc5645e7)
